bucket/kafka.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "2.2.0",
+    "description": "Apache Kafka. A distributed streaming platform.",
+    "homepage": "https://kafka.apache.org/",
+    "license": "Apache-2.0",
+    "url": "https://www.apache.org/dist/kafka/2.2.0/kafka_2.12-2.2.0.tgz",
```

Comment:
What is that `2.12`? Where does it come from?

Reply:
Kafka is written in Scala. `2.12` is the Scala version with which this binary of Kafka was compiled.